### PR TITLE
Improve documentation on custom clinical attributes and fix broken URLs

### PR DESCRIPTION
**Pull request by sandertan**
Changes proposed in this pull request:
- Improve documentation on custom clinical attributes and include the Clinical Data Dictionary
- Fix broken URLs
- Remove reference to specific version of seed database

This PR replaces #4781.

### CODE REVIEW

File: docs/File-Formats.md [hunk 1]
```diff
@@ -1001,7 +1006,7 @@ This will lead to a timeline track for Lab_test with an additional subtrack spec
 
 
 ## Gistic Data
-Running GISTIC 2.0 on e.g. GenePattern not only provides the [Discrete Copy Number Data](#discrete-copy-number-data), but also provides an amp_genes and a del_genes file. These cannot be directly imported into cBioPortal, but first have to be converted to a different file format. Currently, there is no easy way available to do this. However, the cBioPortal team is aiming to make the necessary [cbioportal_pipelines](https://github.com/cBioPortal/cbioportal-pipelines/blob/master/importer/src/main/java/org/mskcc/cbio/importer/converter/internal/GisticGenesConverterImpl.java) functionality available via issue [#873](https://github.com/cBioPortal/cbioportal/issues/873). 
+Running GISTIC 2.0 on e.g. GenePattern not only provides the [Discrete Copy Number Data](#discrete-copy-number-data), but also provides an amp_genes and a del_genes file. These cannot be directly imported into cBioPortal, but first have to be converted to a different file format. An example can be found in the [ACC TCGA study](https://github.com/cBioPortal/datahub/blob/master/public/acc_tcga/data_gistic_genes_amp.txt) on cBioPortal Datahub.
```
Comment (pieterlukasse) [inline docs/File-Formats.md]: Should we deprecate GisticGenesConverterImpl.java in this case? I get from your diff that you find it useless.

Comment (sandertan) [reply docs/File-Formats.md]: It's already deprecated. I can't find this file anywhere. But I've also never needed it, so I think it's safe to remove it from the documentation.

**Comment by sandertan** [comment]
I'll add a commit to change the seed to 2.7.0 when https://github.com/cBioPortal/datahub/pull/377 is merged.

**Comment by sandertan** [comment]
@pieterlukasse I added one additional commit to remove references to a specific version of the seed database. If you agree, can you approve again?

**Comment by jjgao** [comment]
ready to be merged?

**Pull request merged**